src/theme.tsx

```diff
@@ -162,11 +162,12 @@ export type Theme = {
     // TODO: rename this props to navigationBarHeight (or something similar) in next major
     dimensions: {headerMobileHeight: number; headerDesktopHeight: number};
     colorValues: Colors;
+    borderRadii: BorderRadiiConfig;
     textPresets: TextPresetsConfig;
     Link: LinkComponent;
     isDarkMode: boolean;
     isIos: boolean;
     useHrefDecorator: () => (href: string) => string;
-    t: (token: TextToken) => string;
+    t: (token: TextToken, ...params: Array<string | number>) => string;
```

Comment:
Shouldn't params be optional?

Reply:
they are optional. If not passed any additional parameters, the params array will be empty (`[]`)